server/log_test.go

```diff
@@ -147,8 +147,12 @@ func TestLogOutput(t *testing.T) {
 	d := &dummyLogger{}
 
 	checkLogger := func(output string) {
-		if d.msg != output {
-			t.Fatalf("Unexpected logger message: %v", d.msg)
+		checkMsg := output
```

Comment:
After discussion, I realize my mistake since the addition of the prefix is done before delegating to the (dummy) logger.